app/models/find_api/course.rb

```diff
@@ -22,7 +22,7 @@ def subject_codes
     end
 
     def self.fetch(provider_code, course_code)
-      where(recruitment_cycle_year: RecruitmentCycle.current_year)
+      where(year: RecruitmentCycle.current_year)
```

Comment:
I'm a little confused about `year`... Could you explain a bit more where the year information comes from in the existing and the new public APIs?

Reply:
`year` is a parameter name for the JSON API, eg https://api.publish-teacher-training-courses.service.gov.uk/api-reference.html#recruitment_cycles-year-providers-provider_code-courses-get

It's part of the URL — eg in the above example `/recruitment_cycles/{year}/providers/{provider_code}/courses`